src/griffe/finder.py

```diff
@@ -268,8 +269,12 @@ def _extend_from_editable_modules(self):
         for path in self.search_paths:  # noqa: WPS440
             for item in self._contents(path):
                 if item.stem.startswith(("__editables_", "__editable__")) and item.suffix == ".py":
-                    with suppress(UnhandledEditableModuleError):
-                        self._append_search_path(_handle_editable_module(item))
+                    self._extend_from_an_editable_module(item)
+
+    def _extend_from_an_editable_module(self, item: Path):
+        with suppress(UnhandledEditableModuleError):
+            for editable_path in _handle_editable_module(item):
+                self._append_search_path(editable_path)
```

Comment:
Since the function is not reused elsewhere, no need to split it out.

Reply:
Ok, The code complexity checking claims nesting these loops creates a too complex function. Should I silent that warning?

Reply:
Ha, I was wondering if you split it for that reason! Yes, just `# noqa: WPS231` it please :slightly_smiling_face: